frontend/src/App.scss

```diff
@@ -1,10 +1,93 @@
+@import "./styles/prelude.scss";
+
+.error {
+  color: red;
+}
+
 .App {
+  color: silver;
   padding: 20px;
   min-height: 100vh;
-  background-color: lightgray;
+  background-color: $primary-color;
   box-sizing: border-box;
+
+  h1, h3, h4 {
+    color: whitesmoke;
+  }
+
+  hr {
+    margin: 10px 0px 50px;
+  }
 }
 
-.error {
-  color: red;
-}
\ No newline at end of file
+.bi {
+  padding-right: 3px;
+}
+
+.bi-search {
+  color: silver;
+  padding-right: 5px;
+}
+
+.navBar {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-between;
+  align-items: center;
+  box-sizing: border-box;
+  margin: 20px auto;
+
+  padding: 5px;
+
+  h1 {
+    margin: 0;
+
+    a {
+      align-items: center;
+    }
+  }
+
+  #login {
+    display: flex;
+  }
+  
+  a {
```

Comment:
This tag comes up quite frequently in this file, can you leave a comment next to each of them to specify what they are affecting for readability's sake?

Reply:
You would want me to comment in the SCSS file? It's mostly just me tweaking the padding and style of the links (since the `react-router-dom` <Link> tag basically returns <a> tags (which are just links)) so that they all have the same overall style. However, your comment made me realize I had useless code in this file, so it's been removed and now I don't think a comment will be necessary (you'll see what I mean when I push my commits at a later time)

Reply:
I'm good without the comments now, thanks!